alu_u32/src/div/stark.rs

```diff
@@ -0,0 +1,15 @@
+use super::columns::Div32Cols;
+use super::Div32Chip;
+use core::borrow::Borrow;
+
+use p3_air::{Air, AirBuilder};
+use p3_field::{AbstractField, PrimeField};
+use p3_matrix::MatrixRows;
+
+impl<F, AB> Air<AB> for Div32Chip
+where
+    F: PrimeField,
+    AB: AirBuilder<F = F>,
+{
+    fn eval(&self, builder: &mut AB) {}
```

Comment:
nit: let's just add a TODO to avoid any confusion and make sure it can't be forgotten